src/backend/database.js

```diff
@@ -9,6 +9,7 @@ const knex = require("knex")({
     user: process.env.DB_USER,
     password: process.env.DB_PASSWORD,
     database: process.env.DB_NAME,
+    timezone: 'UTC'
```

Comment:
This could be a good addition to the meal-sharing-template. Can you add that line to this repo: https://github.com/HackYourFuture-CPH/meal-sharing-template

You can use the webinterface. Just click on the edit icon and then make a PR with the change

Reply:
Done.